packages/create-block/lib/templates.js

```diff
@@ -59,6 +59,7 @@ const predefinedPluginTemplates = {
 			},
 			viewScript: 'file:./view.js',
 			example: {},
+			folderName: './src/$slug',
```

Comment:
I modified that only for the default templates bundled with the package instead of for every project template, as I'm not entirely sure what implications it would have for developer docs. @ndiego, do you forsee any complications?